SPECS/netdata.spec

```diff
@@ -433,6 +431,7 @@ echo "Netdata go plugin can be easily installed with %{_sbindir}/netdata-install
 * Fri May 24 2024 Thierry Escande <thierry.escande@vates.tech> - 1.44.3-1.1
 - Fix build errors with gcc 4.8
 - Add conditions for protobuf Requires
+- Remove unneeded Requires for nodejs and BuildRequires for httpd and libpfm-devel
```

Comment:
This doesn't mention the added libyaml Requires (why is it needed, by the way?)

Reply:
Fixed